### ISSUE TITLE
Theme level styling for error messages

### ISSUE THREAD

**Issue by benmneb** [bony2023/react-terminal#36]
I know we can style error messages like this `errorMessage={() => <span style={{color: 'red'}}>Command not found</span>}`, but support on a theme level could be useful. I would imagine something would need to be edited around [here](https://github.com/bony2023/react-terminal/blob/4494c5d58a58715df1040c16068a2d6a557ba4c8/src/hooks/editor.tsx#L105-L136) in addition to the theme objects, but I'm not really sure how to go about it myself.

**Comment by benmneb** [comment]
#38 should fix this.

But it would also be nice to allow a function prop to include the errored command in the error message, to replicate real CLI's better, something like:
`errorMessage={(err) => <span style={{color: 'red'}}>Command not found: {err}</span>` would be perfect.

Currently this only returns something if the errored command has a whitespace in it, and it returns everything after the first whitespace, like its being trimmed there or something... ie
```
>>> first second third etc
Unknown command: second third etc
```

**Comment by benmneb** [comment]
In the meantime I've been using something like this, which actually works fine.

```
const themes = {
  ocean: {
    themeBGColor: "#224fbc",
    themeToolbarColor: "#216dff",
    themeColor: "#e5e5e5",
    themePromptColor: "#00e5e5",
    errorColor: "#e5e500",
    linkColor: "#e5e5e5"
  }
  // etc etc
};

const error = {
  color: themes[theme].errorColor,
  fontWeight: "bold"
};
const success = {
  color: themes[theme].successColor,
  fontWeight: "bold"
};

const commands = {
  send: (input) => {
    if (!input) {
      return <span style={error}>Nothing to send</span>;
    }
       
     return <span style={success}>Sent succesfully</span> 
}

<ReactTerminal
    commands={commands}
    errorMessage={<span style={error}>Command not found</span>}
    themes={themes}
    theme={theme}
  />

```

**Comment by bony2023** [comment]
This looks like a great feature. I think we can modify the caller function to include extra terminal-related arguments which can include a bunch of data including the command.

**Comment by bony2023** [comment]
> Support on a theme level could be useful. I would imagine something would need to be edited around here in addition to the theme objects, but I'm not really sure how to go about it myself.

You are absolutely correct. More specifically [here](https://github.com/bony2023/react-terminal/blob/4494c5d58a58715df1040c16068a2d6a557ba4c8/src/hooks/editor.tsx#L116-L118), we need to pass the `command` as well along with `commandArguments`

**Comment by bony2023** [comment]
This can be a quick fix, but more future proof approach would be to go with the extra data passed on to the callback function.
